docs-new/docs/graphql/core/guides/data-modelling/soft-deletes.mdx

```diff
@@ -72,74 +74,69 @@ mutation {
 
 ## Step 3: Set up appropriate insert/update/delete permissions
```

Comment:
Code snippet before this section starting from Line 57 has missed Line highlighting